dvc/compare.py

```diff
@@ -0,0 +1,169 @@
+from itertools import chain, repeat, zip_longest
```

Comment:
Should this file go to dvc/ui or dvc/utils?

Reply:
@efiop, this is not really about UI, most of it is unifying the diff/show which I am thinking might be a part of the unified diff/show that @dberenbaum is proposing, so it's not just `utils` as well. It might be easier later this way. 🙂

Reply:
Though I am open for better name that covers both `show` and `diff`. 😄

Reply:
Ahh, I get the confusion now. #5872 will immediately follow after this, so I am trying to keep these show/diff in a single place.